backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Shop.java

```diff
@@ -0,0 +1,34 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import lombok.Data;
+
+@Data
+public class Shop {
+    private String shopkeeperUid;
+    private String shopOwnerName;
+    private String shopName;
+    private String address;
+    private String phnNo;
+    private Integer lastTokenAlloted;
+    private String shopType;
+    private Integer shopId;
+    private List<UUID> tokens = new ArrayList<>();
+
+    public Shop() {
+    }
```

Comment:
Just wondering if we should use @Noargsconstructor annotation of lombok here and allArgsConstructor Annotation for the below constructor?

Reply:
Done!